docker/ubuntu-reportd/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:22.04
+
+RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
```

Comment:
You should export `DEBIAN_FRONTEND=noninteractive` to avoid having potential debconf prompting (even if this is really unlikely in the base image). If you start docker as a CLI (not the cli/service handler), some terminals were never set properly to non interactive, you get the debconf prompt on upgrade (that we try to avoid on ubuntu, but bugs exists) and then. the whole build is blocked.